packages/web3-contracts/compile-contracts.ts

```diff
@@ -35,11 +35,11 @@ async function main() {
     ])
 
     // format code
-    run(GENERATED_PATH, 'npx', 'prettier', '--write', '*')
+    shell`${GENERATED_PATH}, 'npx', 'prettier', '--write', '*'`
 
     // add to git stage
-    run(ABIS_PATH, 'git', 'add', '.')
-    run(GENERATED_PATH, 'git', 'add', '.')
+    shell`${ABIS_PATH}, 'git', 'add', '.'`
```

Comment:
safe?

Reply:
plz revert this modify.

Reply:
@Jack-Works , why remove `run()` in https://github.com/DimensionDev/Maskbook/commit/e523a09a7682c8cc75af80a2a015ea38384f470e ?

Reply:
@zhouhanseng you should copy `run` from e523a09a7682c8cc75af80a2a015ea38384f470e into this folder. The content in `@masknet/cli` is not designed to be compiled or exported.